gsa/src/web/pages/tlscertificates/component.js

```diff
@@ -57,34 +57,36 @@ class TlsCertificateComponent extends React.Component {
   handleTlsCertificateDownload(cert) {
     const {detailsExportFileName, username, onDownloaded} = this.props;
 
-    const fullCert = loadEntity(cert.id);
-
-    const {
-      creationTime,
-      certificate,
-      entityType,
-      id,
-      modificationTime,
-      name,
-    } = fullCert;
-
-    this.handleInteraction();
-
-    const filename = generateFilename({
-      creationTime,
-      extension: 'pem', // this gets overwritten?
-      fileNameFormat: detailsExportFileName,
-      id,
-      modificationTime,
-      resourceName: name,
-      resourceType: entityType,
-      username,
-    });
-
-    onDownloaded({
-      filename,
-      mimetype: 'application/x-x509-ca-cert',
-      data: create_pem_certificate(certificate),
+    this.props.loadEntity(cert.id).then(response => {
```

Comment:
Does not make much sense to use the action creator function here to load the tls cert. It even would crash if a second request is already running.